config/locales/simple_form.en.yml

```diff
@@ -222,7 +223,8 @@ en:
         setting_expand_spoilers: Always expand posts marked with content warnings
         setting_hide_network: Hide your social graph
         setting_reduce_motion: Reduce motion in animations
-        setting_system_font_ui: Use system's default font
+        setting_system_font_ui: Use operating system default font
```

Comment:
Is the string change here intentional?

Reply:
Only to make them consisntent between the two descriptions. I've changed it not change this one and to format the scrollbar option similarly.